hack/start_flotta_k8s.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export OPERATOR_PATH="<path to flotta operator repo>"
+export DEVICE_WORKER_PATH="<path to flotta device worker repo>"
+export YGGDRASIL_PATH="<path to yggdrasil repo>"
+
+# Start flotta operator
+gnome-terminal --tab -- bash -c \
```

Comment:
I don't think gnome-terminal is the best solution here. Consider `nohup` and smaller script files.
For example:
`nohup  > start-operator.sh >operator.log 2>&1 &`

Reply:
The main idea for using `gnome-terminal` here was to open automatically 3 tabs for the operator, device worker and yggdrasil. If I understand correctly `nohup` is not used for that purpose, no?

If we split this script anyway, another option is to pass the `start-all.sh` script and add to the documentation to run each script (`start-operator.sh` and `start-agent`) from a separate terminal (and then there's no need for `nohup` or `gnome-terminal`). WDYT?

(and BTW, why gnome-terminal isn't the best solution here?)